Plugin/README.txt

```diff
@@ -1,18 +1,20 @@
 Building on any platform:
-1. Get Alembic.
-        Get Alembic releases at https://github.com/alembic/alembic/releases
-2. Get HDF5 (needed for legacy alembic files)
-3. Build the abci plugin
-4. Install it in the AlembicImporter project
-5. Test by dragging an abc file into the project and into the scene, then scrubbing the "time" on the abc file.
 
-To publish you also need:
+1. Get Alembic source code
+        Get Alembic releases at https://github.com/alembic/alembic/releases
+2. Get HDF5 
+        Needed for legacy alembic files
```

Comment:
Probably point people to the build.sh for the dependences. We're missing some here. You need Alembic and HDF5, but also ilmbase and ispc.

Reply:
Or, you don't need any of them because the dependences are included.